tests/virt_autotest/virt_utils.pm

```diff
@@ -545,4 +545,14 @@ sub generate_testcase_xml {
     $xml_writer->endTag('testcase');
 }
 
+#RAW do not support Snapshot, so skip Snapshot test if guest disk type as RAW
+sub check_guest_disk_type {
+    my $guest           = shift;
+    my $guest_disk_type = script_output("virsh dumpxml $guest | grep \"<driver \" | grep -o \"type='.*'\" | cut -d \"'\" -f2");
+    if ($guest_disk_type =~ /raw/) {
+        record_info "INFO", "SKIP Snapshot test if guest disk type as raw";
+        return;
+    }
+}
+
```

Comment:
Finally please 2 verification runs after your changes done, as you did in description verification jobs, 1 for raw disk guest, 1 for !raw disk type.

Reply:
had updated code already
Add the latest verification runs:
gi-guest_developing-on-host_sles15sp2-kvm
http://10.67.19.82/tests/434
gi-guest_developing-on-host_sles11sp4-kvm
http://10.67.19.82/tests/428

Reply:
Why do not I see your change for: 
Finally please 2 verification runs after your changes done, as you did in description verification jobs, 1 for raw disk guest, 1 for !raw disk type.

Reply:
Had updated code as below:
`+    if ($guest_disk_type =~ /raw/) {
+        record_info "INFO", "SKIP Snapshot test if guest disk type as raw";
+        return;
+    }
+    else {
+        if ($guest_disk_type =~ /qcow2/) {
+            record_info "INFO", "Start Snapshot test with the guest disk type as qcow2";
+        }
+    }
`

Reply:
had updated code again. 
Also, add the latest verification runs:
gi-guest_developing-on-host_sles15sp2-kvm
http://10.67.19.82/tests/443
gi-guest_developing-on-host_sles11sp4-kvm
http://10.67.19.82/tests/444